annotations.yaml

```diff
@@ -5839,14 +5839,14 @@ perl-cpan-meta-validator:
     name: perl-cpan-meta-validator
     description: Validate cpan distribution metadata structures
     license: perl_5
-    home_url: http://metacpan.org/pod/CPAN::Meta::Validator
+    home_url: https://github.com/metacpan/metacpan-web/issues
     total_pulls: 38
     manually_check: true
 perl-text-wrap:
     name: perl-text-wrap
     description: line wrapping to form simple paragraphs
     license: perl_5
-    home_url: http://metacpan.org/pod/Text::Wrap
+    home_url: https://github.com/metacpan/metacpan-web/issues
     total_pulls: 38
     manually_check: true
```

Comment:
the cpan repository can not be used for all the cpan packages. please replace that.

Reply:
ok , i'll replace it